components/Chart/chart.mixin.js

```diff
@@ -0,0 +1,62 @@
+export default {
+  props: {
```

Comment:
these props are not used by the mixin but in the components so they should be moved to the component except if the same will be reused for other charts. So up to you.